1.10/networking/index.md

```diff
@@ -31,7 +31,7 @@ Spartan acts as a DNS masquerade for Mesos DNS on each agent. The Spartan instan
 The Spartan instance on each agent also acts as a DNS server for any service that is load balanced using the DC/OS internal load balancer called [Minuteman](/docs/1.10/networking/mesos-dns/). Any service that is load balanced by Minuteman gets a [virtual-ip-address (VIP)](/docs/1.10/networking/mesos-dns/) and an FQDN in the `"*.l4lb.thisdcos.directory"` domain. The FQDN allocated to a load-balanced service is then stored in Spartan. All Spartans instances exchange the records they have discovered locally from Minuteman by using GOSSIP. This provides a highly available distributed DNS service for any task that is load balanced by Minuteman. For more information, see the [Spartan repository](https://github.com/dcos/spartan).
 
 # Load Balancing
-East-west load balancing is provided by Minuteman. North-south load balancing is provided by [Marathon LB](/docs/1.10/networking/marathon-lb/). Marathon-LB is based on HAProxy, a rapid proxy and load balancer. It is installed as a DC/OS Universe package.
+East-west load balancing is provided by Minuteman. North-south load balancing is provided by [Marathon-LB](/docs/1.10/networking/marathon-lb/) and Edge-LB (Entprise only). Marathon-LB is based on HAProxy, a rapid proxy and load balancer. Both Marathon-LB and Edge-LB are installed as DC/OS Universe packages.
```

Comment:
both marathon-lb and edge-lb can provide east-west as well.

Not sure the goal here is provide an in-depth comparison, but it's as follows:
- minuteman: a "L4" (aka "Layer 4" or "Transport Layer") load balancer. It can only do east-west.
- Marathon-lb/edge-lb: primarily a "L7" (aka "Layer 7" or "Application Layer") load balancer, but is also capable of "L4" load balancing (but less performant than minuteman). It's primarily for north-south, but can also be configured to do east-west.